modules/data.atmosphere/R/download.NOAA_GEFS.R

```diff
@@ -234,17 +255,16 @@ download.NOAA_GEFS <- function(outfolder, lat.in, lon.in, sitename, start_date =
   #For each ensemble
   for (i in 1:21) { # i is the ensemble number
     #Generating a unique identifier string that characterizes a particular data set.
-    identifier = paste("NOAA_GEFS", sitename, i, format(start_date, "%Y-%m-%dT%H:%M"), 
-                       format(end_date, "%Y-%m-%dT%H:%M"), sep=".")
-    
-    ensemble_folder = file.path(outfolder, identifier)
+    identifier = paste("NOAA_GEFS", site_id, i, format(start_date, "%Y-%m-%dT%H:%M"), 
+                       format(end_date, "%Y-%m-%dT%H:%M"), sep="_")
     
+
     #Each file will go in its own folder.
-    if (!dir.exists(ensemble_folder)) {
-      dir.create(ensemble_folder, recursive=TRUE, showWarnings = FALSE)
+    if (!dir.exists(outfolder)) {
+      dir.create(outfolder, recursive=TRUE, showWarnings = FALSE)
```

Comment:
I understand the argument for not putting ever ensemble in its own folder for GEFS, but you should take a careful look at how this impacts the logic of the ensemble met code. I worry this could lead to models being confused about which files to use (e.g. some models or modules might not be checking filenames carefully enough to distinguish among 21 different files for the same date-range in the same folder) and also might have been done to maintain generality with other ensemble met options that were longer (e.g. PalEON) where you definitely wouldn't want >100 met ensembles with 1160 annual files each all in the same folder. I think the 'every ensemble member in it's own folder' norm was done to minimize the risk of mistakes/confusion.

Reply:
I checked it with my forecasts before I made the pull request and the met worked properly with SIPNET. As in it properly created ensemble clim files and each of those go into their own ensemble directory. I also looked through other people's met files that they have downloaded and couldn't find other examples of the nested directories for ensembles

Reply:
Well, AFAIK the only other ensemble met options right now are ERA5 and PalEON.  Just took a look at the ERA5 code, and it is definitely splitting ensemble members into different folders https://github.com/PecanProject/pecan/blob/develop/modules/data.atmosphere/R/met2cf.ERA5.R#L131. PalEON is mostly generated outside of PEcAn and then registered posthoc.

Reply:
I think Hamze might have based the ensemble part of the ERA5 code off of the GEFS download. I remember him asking me about the code a while back and it the ERA code looks identical to GEFS there. I can change it back if that's what you would rather but it did work normally with my code and was able to make the properly clim files. My thought process was  that it's just what Luke thought would be good for ensembles, not necessarily something with the pecan standards since we dont have other examples of ensemble met downloads.

Reply:
Yes let's put it back. And yes, Luke's work was the first addition of ensembles (so in some sense it defined the standard), but he also did ask me questions 20 times a day, so it wasn't a rogue decision.